### PR TITLE
Register Deldir: v0.0.1, v0.0.2

### PR DESCRIPTION
**Pull request by robertdj**
Delaunay/Voronoi computations with capabilities that I haven't found in similar *Julia* packages.

### CODE REVIEW

**Comment by tkelman** [comment]
As I mentioned on the mailing list, to be on the explicit side it would be best to put the full GPL license in your license file with a reference to the fact that it applies to the Fortran code you've included under deps.

**Comment by robertdj** [comment]
I'll update the license file.

**Comment by robertdj** [comment]
Can you send me a link to the mailing list for future reference?

**Comment by tkelman** [comment]
https://groups.google.com/forum/#!forum/julia-users you mean?

**Comment by robertdj** [comment]
Sorry, I misunderstood what you meant by mailing list. I've pushed an updated license file and removed the `Pkg.dir`.

**Comment by tkelman** [comment]
Looks good. You can either add a new 0.0.3 tag into this same branch of your metadata fork and it will automatically update the PR, or you can wait until this is merged and do the new tag separately. I'm going to leave this open for ~24 hours of other review comments, and merge if there are none by then.

**Comment by robertdj** [comment]
Great, thanks!

**Comment by tkelman** [comment]
wrong button?

File: Deldir/url [hunk 1]
```diff
@@ -0,0 +1 @@
+https://github.com/robertdj/Deldir.jl
```
Comment (tkelman) [inline Deldir/url]: oh, apparently this has to end in `.git` - you can edit in place with the pencil icon at https://github.com/robertdj/METADATA.jl/blob/metadata-v2/Deldir/url

**Comment by robertdj** [comment]
Yes, my bad with the closing! I've corrected the url.

**Pull request merged**